app/assets/stylesheets/application.scss

```diff
@@ -17,3 +17,7 @@ $govuk-font-url-function: "font-url";
 .govuk-phase-banner{
   margin-bottom: 25px;
 }
+
+.text-right{
+  text-align: right;
+}
```

Comment:
Definitely sure nothing in the frontend styling we import that gives us this?

Reply:
About 80% - had a look at some of the other admin tools and this seems to be what they were doing. Had a quick look through the Design System too, no mention of it.

Reply:
Okay I see